docs/src/main/sphinx/release/release-365.md

```diff
@@ -0,0 +1,126 @@
+# Release 365 (XX Dec 2021)
+
+## General
+
+* Add support for [`TRUNCATE TABLE`](/sql/truncate). ({issue}`8921`)
+* Add support for time travel queries. ({issue}`8773`)
+* Add [`contains`](ip_address_contains) function to check whether a CIDR contains an IP address. ({issue}`9654`)
+* Report connector metrics in `EXPLAIN ANALYZE VERBOSE`. ({issue}`9858`)
+* Report operator input distribution in ``EXPLAIN ANALYZE VERBOSE``. ({issue}`10133`)
+* Allow executing ``INSERT`` or ``DELETE`` statements on tables restricted with a row filter. ({issue}`8856`)
+* Improve performance of inequality joins. ({issue}`9307`)
+* Improve performance of joins with small build side. ({issue}`9851`)
+* Improve CPU utilization by adjusting ``task.concurrency`` automatically for number of physical cores. ({issue}`10088`)
+* Make query final statistics more accurate. ({issue}`9888`, {issue}`9913`)
+* Improve query planning performance for queries containing large `IN` predicates. ({issue}`9874`)
+* Fix incorrect results when casting `bigint` values to `varchar(n)` type. ({issue}`552`)
+* Fix ``PREPARE`` statements for ``DROP/INSERT`` when table or schema name contains special characters. ({issue}`9823`)
+* Fix minor memory leak when queries are abandoned during the initial query submission phase. ({issue}`9962`)
+
+## Security
+
+* Allow configuring HTTP proxy for OAuth2 authentication. ({issue}`9920`, {issue}`10069`)
+* Add group-based and owner-based query access rules to file based system access control. ({issue}`9811`)
+* Use internal names for discovery client when automatic TLS is enabled for internal communications. This allows `discovery.uri` 
+  to be configured using a normal DNS name like `https://coordinator.trino`, and still use automatic TLS certificates. ({issue}`9821`)
+* Use Kerberos operating system ticket cache if keytab file is not provided in JDBC and CLI for Kerberos authentication. ({issue}`8987`)
+* Fix internal communication automatic TLS on Java 17. ({issue}`9821`)
+
+## BigQuery connector
+
+* Support reading `bignumeric` type whose precision is less than or equal to 38. ({issue}`9882`)
+* Fix failure when a schema is dropped during listing tables. ({issue}`9954`)
+
+## Cassandra connector
+
+* Support reading user defined types in Cassandra. ({issue}`147`)
+
+## ClickHouse connector
+
+* Add support for [truncating tables](/sql/truncate). ({issue}`8921`)
+* Fix incorrect query results when query contains predicates on `real` type columns. ({issue}`9998`)
+
+## Druid connector
+
+* Add support for [truncating tables](/sql/truncate). ({issue}`8921`)
+
+## Elasticsearch connector
+
+* Add support for additional Elastic Cloud node roles. ({issue}`9892`)
+* Fix failure when empty values exist in numeric fields. ({issue}`9939`)
+
+## Hive connector
+
+* Allow reading empty files. ({issue}`9929`)
```

Comment:
so this was failing previously? Or we just wasted some time on processing useless splits?

Reply:
cc @Praveen2112

Reply:
This is actually two changes:

* Allow reading empty files of type Parquet, RCFile, SequenceFile. (and maybe Avro ... previously allowed for ORC and textual formats)
* Improve performance by not generating splits for empty files.